src/client/initialize-app.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+import emptyFunction from '../utils/empty-function';
+
+function initializeShopifyPlatform(context, options, hull) {
+  const { customerId, accessToken } = options;
+
+  if (/^[0-9]+$/.test(customerId) && !accessToken) {
+    hull.api.put('services/shopify/customers/' + customerId).then(function() {
+      document.location.reload();
+    });
+  }
+
+  Hull.on('hull.user.logout', function() {
+    document.location = '/account/logout';
+  });
+}
+
+function getAppInitializer(app) {
+  if (app.type === 'platforms/shopify_shop') {
```

Comment:
no big deal, Thought this would be cleaner since we're retiring the 'app' concept from public visiblity